packages/svelte/src/internal/client/render.js

```diff
@@ -1863,6 +1845,7 @@ function await_block(anchor_node, input, pending_fn, then_fn, catch_fn) {
 			return;
 		}
 		const transitions = render.transitions;
+		remove_in_transitions(transitions);
```

Comment:
how come we added this here and on line 1994? was there a bug before?

Reply:
I just noticed we weren't doing it lol. We should have been.